sycl/source/detail/program_manager/program_manager.cpp

```diff
@@ -2118,6 +2129,28 @@ std::pair<RT::PiKernel, std::mutex *> ProgramManager::getOrCreateKernel(
                         &(BuildResult->MBuildResultMutex));
 }
 
+bool isDevSupportImgAspects(std::set<RTDeviceBinaryImage *> BinImages,
+                            const device &Dev) {
+  for (RTDeviceBinaryImage *Img : BinImages) {
```

Comment:
I would have preferred to have one of `std::all_of`/`std::any_of`/`std::none_of` here.